src/rules.c

```diff
@@ -130,8 +139,13 @@ void SchemaPrefixes_Create() {
   ScemaPrefixes_g = NewTrieMap();
 }
 
+static void freePrefixNode(void *ctx) {
+  SchemaPrefixNode *node = ctx;
+  array_free_ex(node->index_specs, NULL);
```

Comment:
Why not just array_free if you give a NULL free func?

Reply:
`array_free` is a static function